src/cui.cpp

```diff
@@ -64,6 +64,9 @@ const char *COLUMN_FORMAT_PID = "%7d";
 const char *COLUMN_FORMAT_SENT = "%11.3f";
 const char *COLUMN_FORMAT_RECEIVED = "%11.3f";
 
+// All descriptions are padded to 6 characters in length with spaces
+const char* const desc_view_mode[VIEWMODE_COUNT] = {"KB/sec", "MB/sec", "GB/sec", "KB    ", "B     ", "MB    "};
```

Comment:
I think I understand what you mean now, trace mode. Since the enumeration values are exposed to the user, changing them would be a breaking change. It would have to be appended then. Gui users can still start it in the mode they want without having to deal with the cycling.

Reply:
OK. Merge proposal has been updated with the new modes appended. Man page and cli help updated accordingly.